arcade/examples/sprite_health.py

```diff
@@ -1,230 +1,249 @@
-
-        # Create the coins
-        for i in range(COIN_COUNT):
-
-            # Create the coin instance
-            # Coin image from kenney.nl
-            coin = SpriteWithHealth(":resources:images/items/coinGold.png",
-                                    SPRITE_SCALING_COIN,
-                                    max_health=5)
-
-            # Position the coin
-            coin.center_x = random.randrange(SCREEN_WIDTH)
-            coin.center_y = random.randrange(150, SCREEN_HEIGHT)
-
-            # Add the coin to the lists
-            self.coin_list.append(coin)
+    def setup(self) -> None:
+        """Set up the game and initialize the variables."""
+        # Setup player and enemy positions
+        self.player_sprite.position = self.width // 2, self.height // 4
+        self.enemy_sprite.position = self.width // 2, self.height // 2
 
         # Set the background color
-        arcade.set_background_color(arcade.color.AMAZON)
-
-    def on_draw(self):
-        """
-        Render the screen.
-        """
+        self.background_color = arcade.color.AMAZON
 
-        # This command has to happen before we start drawing
+    def on_draw(self) -> None:
+        """Render the screen."""
+        # Clear the screen. This command has to happen before we start drawing
         self.clear()
 
-        # Draw all the sprites.
-        self.coin_list.draw()
+        # Draw all the sprites
+        self.player_sprite.draw()
+        self.enemy_sprite.draw()
         self.bullet_list.draw()
-        self.player_list.draw()
+        self.bar_list.draw()
 
-        for coin in self.coin_list:
-            coin.draw_health_number()
-            coin.draw_health_bar()
+    def on_mouse_motion(self, x: float, y: float, dx: float, dy: float) -> None:
+        """Called whenever the mouse moves."""
+        self.player_sprite.position = x, y
 
-        # Render the text
-        arcade.draw_text(f"Score: {self.score}", 10, 20, arcade.color.WHITE, 14)
+    def on_update(self, delta_time) -> None:
+        """Movement and game logic."""
+        # Check if the player is dead. If so, exit the game
+        if self.player_sprite.health <= 0:
+            arcade.exit()
 
-    def on_mouse_motion(self, x, y, dx, dy):
-        """
-        Called whenever the mouse moves.
-        """
-        self.player_sprite.center_x = x
+        # Increase the enemy's timer
+        self.enemy_timer += delta_time
 
-    def on_mouse_press(self, x, y, button, modifiers):
-        """
-        Called whenever the mouse button is clicked.
-        """
-        # Gunshot sound
-        arcade.play_sound(self.gun_sound)
-        # Create a bullet
-        bullet = arcade.Sprite(":resources:images/space_shooter/laserBlue01.png", SPRITE_SCALING_LASER)
+        # Update the player's indicator bar position
+        self.player_sprite.indicator_bar.position = self.player_sprite.center_x, self.player_sprite.center_y + INDICATOR_BAR_OFFSET
```

Comment:
This can be split across multiple lines to make it legible in the web view:
```suggestion
self.player_sprite.indicator_bar.position =\
    self.player_sprite.center_x,\
    self.player_sprite.center_y + INDICATOR_BAR_OFFSET
```

Reply:
Take einarf's advice from discord over my initial suggestion on this: the parentheses are probably the more elegant option.